README.md

```diff
@@ -85,6 +85,15 @@ Example manifest:
 }
 ```
 
+### input
```

Comment:
As `input` is quite an ambiguous property name, especially in the context of Rollup. Can you change this to something more specific, such as `manifestInput`, `manifestKey` or `manifestProperty`?